src/components/organisms/plans/plans.html

```diff
@@ -0,0 +1,55 @@
+<div class="plans">
+  <div class="message">
+    <div class="message-inner">
+      <p class="message-title">ライフスタイルに合わせて選べる多彩なプラン。</p>
+      <p class="small">
+        平日のみプラン、仕事帰りをコアタイムとした夕方以降が中心のプラン、休日をメインとしたプランなど、
+        <br />多彩なプランの中からそれぞれのお客様のライフスタイルに合わせてお選びいただけます。
+        <br />ビジター料金でのご利用も可能ですので、月額会員登録の前に体験をしたいという方もぜひご利用ください。
+      </p>
+    </div>
+  </div>
+  <div class="plan-lists">
+    <ul class="plan">
+      <li class="header-box">デイタイムプラン</li>
+      <li class="box-list1">
+        <div class="box-list1-text">¥5,000</div>
+        <div class="box-list1-text-bottom">月額</div>
+      </li>
+      <li class="box-list2">入会金0円</li>
+      <li class="box-list2">平日：9時～17時</li>
+      <li class="box-list2">土日祝：利用ナシ</li>
+      <li class="box-list2">コアタイムでの利用に最適なプラン</li>
+    </ul>
+
+    <ul class="plan">
+      <li class="header-box">プランテキスト</li>
+      <li class="box-list1">
+        <div class="box-list1-text">¥5,000</div>
+        <div class="box-list1-text-bottom">月額</div>
+      </li>
+      <li class="box-list2">入会金0円</li>
+      <li class="box-list2">平日：9時～17時</li>
+      <li class="box-list2">土日祝：利用ナシ</li>
+      <li class="box-list2">コアタイムでの利用に最適なプラン</li>
+    </ul>
+
+    <ul class="plan">
+      <li class="header-box">プランテキスト</li>
+      <li class="box-list1">
+        <div class="box-list1-text">¥5,000</div>
+        <div class="box-list1-text-bottom">月額</div>
+      </li>
+      <li class="box-list2">入会金0円</li>
+      <li class="box-list2">平日：9時～17時</li>
+      <li class="box-list2">土日祝：利用ナシ</li>
+      <li class="box-list2">コアタイムでの利用に最適なプラン</li>
+    </ul>
+  </div>
+
+  <div class="module">
+    <div class="atoms-button">
+      <a href="#" class="url">プラン料金</a>
+    </div>
+  </div>
```

Comment:
```suggestion
  <div class="button-wrapper">
    <div class="atoms-button">
      <a href="#" class="url">プラン料金</a>
    </div>
  </div>
```
`module` だとクラス名が汎用的すぎて、クラス名だけ見たときにどのような役割を持っているかがわかりません。また、汎用的・抽象的なクラス名だと他の部分で同様の命名をしているクラスが発生しやすく、その分スタイルの衝突も発生しやすくなります。

クラス名の付け方については、以下の記事あたりが参考になるかと思います。
https://pengi-n.co.jp/blog/css-class/